pkgs/tools/misc/zf/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zig,
+  testers,
+  zf,
+  ...
+}:
+stdenv.mkDerivation rec {
+  pname = "zf";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "natecraddock";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-MzlSU5x2lb6PJZ/iNAi2aebfuClBprlfHMIG/4OPmuc=";
+  };
+
+  nativeBuildInputs = [zig];
+
+  buildPhase = ''
+    runHook preBuild
+
+    export HOME="$TMPDIR"
+    zig build -Drelease-safe
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp zig-out/bin/zf $out/bin
+
+    runHook postInstall
+  '';
+
+  passthru.tests.version = testers.testVersion {package = zf;};
+
+  meta = with lib; {
+    homepage = "https://github.com/natecraddock/zf";
+    description = "A commandline fuzzy finder that prioritizes matches on filenames";
+    license = licenses.mit;
+    platforms = platforms.linux;
```

```suggestion
    platforms =  platforms.linux ++ platforms.darwin;
```

This builds and runs just fine on darwin too.

---

right, this was copied over from the other PR. I'm just going to drop the platforms for now instead.